doc/classes/ColorPicker.xml

```diff
@@ -125,6 +137,9 @@
 		<constant name="SHAPE_OKHSL_CIRCLE" value="3" enum="PickerShapeType">
 			HSL OK Color Model circle color space.
 		</constant>
+		<constant name="SHAPE_NONE" value="4" enum="PickerShapeType">
+			Hides the color picker.
```

Comment:
This description is very misleading. Better:
```suggestion
			The color space shape is hidden.
```
(consistent with `picker_shape` property)